control_plane/src/local_env.rs

```diff
@@ -81,6 +93,10 @@ impl Default for PageServerConf {
             http_port: 0,
             auth_type: AuthType::Trust,
             auth_token: "".to_string(),
+            gc_period: "100s".to_string(),
+            gc_horizon: 64 * 1024 * 1024,
+            checkpoint_period: "1s".to_string(),
+            checkpoint_distance: 256 * 1024 * 1024,
```

Comment:
Why not use DEFAULT_CHECKPOINT_PERIOD and other `defaults` here?

Reply:
1 sec and 256Mb seems to be default checkpoint values.
I just prefer to specify them explicitly in this file, because setting for tests may be different with one used in productions. So not want that checkpoint interval for tests depends on default setting for production.